default_config/dev/config.yaml

```diff
@@ -1,8 +1,19 @@
 API_HOST_URL: 'http://localhost:8000'
+SSO_URL: 'http://localhost:8000'
+SSO_URL_SUCCESSFUL_LOGIN:  'http://localhost:8000'
```

Comment:
What do you think about renaming `SSO_URL_SUCCESSFUL_LOGIN` to a more general `SSO_REDIRECT_URL`?

Reply:
I think it means `SSO_FINISHED_URL` which is the URL we read the cookie from.

Reply:
✅
 I used SSO_FINISHED_URL, please let me if you prefer SSO_REDIRECT_URL